run/encdatavault2john.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# Helper script for cracking ENCSecurity DataVault.
+#
+# This software is Copyright (c) 2021, Kudelski Security <sylvain.pelissier at kudelskisecurity.com>
+# and it is hereby released to the general public under the following terms:
```

Comment:
Incidentally, the script is copyright by your company/employer and the C code by you personally - I guess that's deliberate? That's no problem, just pointing out in case it was in error.

Reply:
It was an error thank you.